test/examples_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+const examplesDir = ".."
+const scriptDir = "scripts"
+
+func TestCoinPricesFromCoingecko(t *testing.T) {
+	projectName := "coin_prices_from_coingecko"
+	projectDir := filepath.Join(examplesDir, projectName)
+	requiredEnvVars := []string{
+		"YOUR_COINGECKO_API_KEY",
+	}
+
+	NewProjectTest(t, projectDir).
```

Comment:
All testscript tests occur in a temporary working directory. These commands specify test and WorkDir setup actions that need to take place. For example, all "copy" and "render" commands move the specified file to the workdir.